roboteam_ai/src/coach/defence/DefencePositionCoach.cpp

```diff
@@ -332,7 +332,7 @@ std::shared_ptr<double> DefencePositionCoach::pickNewPosition(const Line &line,
     //search a position on the line on which we can position.
     double lineLen = (line.second - line.first).length();
     double minSkipDist = 0.05;
-    double skipDist = max(minSkipDist, lineLen/30.0);
+    double skipDist =  lineLen/30.0;
```

Comment:
magic

Reply:
Changed this function around a bit, there was some legacy stuff in here.